### ISSUE TITLE
Results of Google Colab's example

### ISSUE THREAD

**Issue by jdiazram** [krrish94/nerf-pytorch#37]
Hi, nice work with NeRF, but I have a question: 
I try to run the example on https://colab.research.google.com/drive/1L6QExI2lw5xhJ-MLlIwpbgf7rxW7fcz3, but even with 92.450 iter the result is close to 0.17 to Val Loss and 7.6 to PSNR without images for coarse and fine. So, why did this happen? or did I lose something?  Adjunct an image with this.

<img width="622" alt="Captura de Pantalla 2022-07-19 a la(s) 09 42 44" src="https://user-images.githubusercontent.com/45439493/179779644-f2879d84-0a0b-4973-beb3-974599d2353f.png">

Another thing I did was uncommenting ReplicateNeRFModel, because another error appeared with this; 

<img width="1404" alt="Captura de Pantalla 2022-07-20 a la(s) 13 13 36" src="https://user-images.githubusercontent.com/45439493/180043250-5425b0a4-0613-4cec-ba41-72f7a9070c4b.png">

Only with this action did the example run

```
model_coarse = ReplicateNeRFModel(
    hidden_size=128,
    num_encoding_fn_xyz=num_encoding_fn_xyz,
    num_encoding_fn_dir=num_encoding_fn_dir,
    include_input_xyz=include_input_xyz,
    include_input_dir=include_input_dir
)
# model_coarse = VeryTinyNeRFModel()
model_coarse.to(device)

# Initialize a fine-resolution model, if specified.
model_fine = ReplicateNeRFModel(
    hidden_size=128,
    num_encoding_fn_xyz=num_encoding_fn_xyz,
    num_encoding_fn_dir=num_encoding_fn_dir,
    include_input_xyz=include_input_xyz,
    include_input_dir=include_input_dir
)
# model_fine = VeryTinyNeRFModel()
model_fine.to(device)
```

Thx

**Comment by Cospui** [comment]
I also encountered this problem. It seems to be caused by bad initialization and sampling or ReLU so the training is stuck in local optima. Changing the seed (e.g., uncomment `seed=1234`) will work

**Comment by nwangfw** [comment]
> I also encountered this problem. It seems to be caused by bad initialization and sampling or ReLU so the training is stuck in local optima. Changing the seed (e.g., uncomment `seed=1234`) will work

Thanks.  How did you figure this out? You are amazing.
